documentation/modules/rn-2.5.adoc

```diff
@@ -0,0 +1,199 @@
+// Module included in the following assemblies:
+//
+// * documentation/doc-Release_notes/master.adoc
+
+[id="rn-2.5_{context}"]
+= {project-full} 2.5
+
+You can use {the-lc} {project-first} to migrate virtual machines from the following source providers to {virt} destination providers:
+
+* VMware vSphere
+* {rhv-full} ({rhv-short})
+* {osp}
+* Open Virtual Appliances (OVAs) that were created by VMware vSphere
+* Remote {virt} clusters
+
+The release notes describe technical changes, new features and enhancements, and known issues.
+
+[id="technical-changes-25_{context}"]
+== Technical changes
+
+This release has the following technical changes:
+
+.Migration using OpenStack
```

Comment:
and there's something confusing about this title since migrations from openstack were introduced in 2.4 - maybe "Migration from OpenStack becomes fully-supported"?

Reply:
changed